app/js/app/services/Api.js

```diff
@@ -272,9 +272,8 @@ define([], function() {
             },
             'getAssignedGroups' : {
                 method: 'GET', 
-                isArray: true,
-                url: urlPrefix + "/assignments/assign/:gameId", 
-                params: {gameId: '@gameId'}
+                //isArray: true,
+                url: urlPrefix + "/assignments/assign/groups?gameboard_ids=:gameboard_ids"
```

Comment:
Is this the best way of doing this? I assume we need this to avoid multiple trips to the API to check each board?